tests/parser/types/test_dynamic_array.py

```diff
@@ -1277,6 +1277,70 @@ def foo(x: uint8) -> uint8:
     assert_tx_failed(lambda: c.foo(241))
 
 
+def test_list_of_nested_struct_arrays(get_contract):
```

Comment:
are these functional tests required for this PR? or just addl dynarray tests?

Reply:
these are additional array tests. should I also add tests for the generated ABI?